build/configs/qemu/tc_network_16m/defconfig

```diff
@@ -0,0 +1,932 @@
+#
+# Automatically generated file; DO NOT EDIT.
+# TinyAra Configuration
+#
+CONFIG_SYSTEM_IPERF=y
```

Comment:
What is this? Definitely you didn't make this through menuconfig. It can cause many problem. Let's update this through menuconfig again.